src/test/resources/keypair.txt

```diff
@@ -1,4 +1,4 @@
-publicKey : [B@6ab778a
-publicKeyHex : 6C0816523644DD538B756F88D5043C311764D44221926E91A189FDA20B96D491
-privateKey : [B@4e3958e7
-privateKeyHex : C8EAF5C3A736BC3B6E5F53B48FC378A0B0165D4428CD06B67C96F4479FD5C14D
\ No newline at end of file
+publicKey : [-96, 35, -78, -46, 67, -117, -82, -31, 51, -87, -51, 81, 80, 118, 20, -70, -128, -32, 117, 82, -20, 35, -114, -80, -119, 12, -53, 62, -25, -115, -9, -110]
```

Comment:
and now it is array of bytes and not hexstring :)

Reply:
I know) Look carefully at the code, there first displays the key in the form of bytes, and then in the form of a hex string

Reply:
Now I see. This is okay.